### ISSUE TITLE
Openmodal_UFF

### ISSUE THREAD

**Issue by MisureMeccaniche** [openmodal/OpenModal#43]
Hi everybody,
I install Openmodal on my notebook (Windows 7 PRO, 64 bits) where Python (3.7.1) is also installed.
When I start Openmodal software it open successfully but DOS Windows show some error:
![openmodal](https://user-images.githubusercontent.com/45893564/50041059-cf7c7500-004e-11e9-8b6c-f47b47688004.png)

When software is open I can create a geometry but cannot import any uff, also the one inside data folder of pyuff-master.

Can you help me please?

Kind regards.

P.S. NI drivers are not installed but I'm not interested on it and I do not have such driver to install.

**Comment by miha-pirnat** [comment]
Yes, I've noticed that uff files from pyuff cannot be imported right now into openmodal. I think this is due to some updates in pyuff, which are not yet in 'exe' version of openmodal. This will be corrected in next version of openmodal.

In the mean time I've uploaded an example dataset here:
https://github.com/openmodal/OpenModal/blob/master/openmodal_test_data.unv

You can import it into openmodal, switch to geometry window, click ''EMApaketov 1.0'' and you should see some nodes (look below).
![slika](https://user-images.githubusercontent.com/11046478/50384922-79ea3d00-06cc-11e9-8409-145a896f64df.png)

**Comment by MisureMeccaniche** [comment]
Hi Miha, this file is working.
It contain geometry and data (FRF): it looks more an Openmodal Project than a real uff.
Do you have an example for only data?
Is it possible to import time history or Openmodal accept only FRF data?
Kind regards.

**Comment by miha-pirnat** [comment]
It is real UFF data, actualy it's an export from LMS test lab.

At this point you can import FRF only.

**Comment by MisureMeccaniche** [comment]
Hi Miha,
I follow your instruction step by step but table, inside animation tab, remain empty: see screenshot attached.

Kind regards.

![animation](https://user-images.githubusercontent.com/45893564/50446965-37b53b80-0918-11e9-938b-a53c62ef6265.png)

**Comment by miha-pirnat** [comment]
Yes, you have to switch to raw measurements to animate or first analyze data under analysis tab and then animate identified modes.
